NeuroID/NeuroIDClass/Extensions/NIDUser.swift

```diff
@@ -124,6 +124,12 @@ public extension NeuroID {
     }
 
     static func setRegisteredUserID(_ registeredUserID: String) -> Bool {
+        if !NeuroID.registeredUserID.isEmpty, registeredUserID != NeuroID.registeredUserID {
+            NeuroID.saveEventToLocalDataStore(NIDEvent(level: "warn", m: "Multiple Registered User Id Attempts"))
+            NIDLog.e("Multiple Registered UserID Attempt: Only 1 Registered UserID can be set per session")
+            return false
+        }
+
```

Comment:
is `NeuroID.registeredUserID` reset to the empty string where appropriate (new session?)?

Reply:
ref: https://github.com/Neuro-ID/trackjs/blob/main/src/collection/collector-api.js#L951

Reply:
Correct, when stop/stopSession is called we clear all session variables (userID, registeredUserID, and linkedSiteID)